### PR TITLE
Support passing tag

### LINKED ISSUE #60: Support tags other than "latest" in pushes

### PR DESCRIPTION
**Pull request by rainersigwald**
Instead of hardcoding "latest" everywhere.

Fixes #60.

### CODE REVIEW

File: System.Containers/LocalDocker.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace System.Containers;
 
 public class LocalDocker
 {
-    public static async Task Load(Image x, string name, string baseName)
+    public static async Task Load(Image x, string name, string tag, string baseName)
```
Comment (BenVillalobos) [inline System.Containers/LocalDocker.cs:11]:
```suggestion
    public static async Task Load(Image x, string name, string baseName, string tag = "latest")
```
maybe?

Comment (rainersigwald) [reply System.Containers/LocalDocker.cs:11]: I don't hate this but who is going to call it? It was easy enough to require it to be explicitly passed for all the uses I found (so far) + easier to fix.

Comment (BenVillalobos) [reply System.Containers/LocalDocker.cs:11]: so on my end if I have no tag to supply ($(Version) was empty or something), should I manually pass `latest`?

Comment (baronfel) [reply System.Containers/LocalDocker.cs:11]: Yeah, that's a safe convention.

**Review by baronfel — APPROVED** [review]

**Pull request merged**